src/Framework/Runner/Interfaces.cs

```diff
@@ -85,6 +85,7 @@ public interface IRunnerSetupData
         bool IsTesting { get; set; }
         string ExcludedCategory { get; set; }
         int SelectedProduct { get; set; }
+        string AdditionalResolutionDirectories { get; set; }
```

Comment:
This is ";" separated list of path into one string, so that OptionSet can work seamlessly.